Tests failing after skimage release

## 🐛 Bug

I think we've got some failing tests after the latest skimage release around data loading . Maybe @jni or @grlee77 can look in and fix. thanks!!!

```python-traceback
  _________________ ERROR at setup of test_add_multichannel_tiff _________________
  
      @pytest.fixture
      def single_tiff():
  >       return [image_fetcher.fetch('data/multipage.tif')]
  
  napari/conftest.py:264: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  data_name = 'multipage.tif'
  
      def fetch(data_name):
          if data_name.startswith("data/"):
              data_name = data_name[5:]
          path = os.path.join(data_dir, data_name)
          if not os.path.exists(path):
  >           raise ValueError(
                  f"Legacy skimage image_fetcher cannot find file: {path}"
              )
  E           ValueError: Legacy skimage image_fetcher cannot find file: /home/runner/.cache/scikit-image/0.19.0/data/multipage.tif
  ```

see https://github.com/napari/napari/runs/4413703143?check_suite_focus=true for example

--- a/setup.cfg
+++ b/setup.cfg
@@ -104,7 +104,7 @@ testing =
     xarray
     fsspec
     zarr
-    scikit-image>=0.18.1
+    scikit-image>=0.18.1,!=0.19.0
     pooch>=1.3.0
     semgrep
     tensorstore>=0.1.13 ; python_version < '3.10'

